Snap.xcodeproj/project.pbxproj

```diff
@@ -702,9 +758,19 @@
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
+				ENABLE_BITCODE = NO;
```

Comment:
Is it there any reason for Carthage to set disable the bitcode?

Reply:
Bitcode is not supported by `XCTest`

Reply:
oh, it makes total sense 😅